### DISCUSSION TITLE
Github mirror for Godot 4 beta versions

### DISCUSSION THREAD

**Discussion by Vitorgus** [godotengine/godot-proposals#6054] [General Discussions]
I've recently downloaded beta 10, but through the default download repository it took VERY long for me. Download speed was always a few Kbps. And there are no tags for the godot 4 betas in the github repository, so there was no other official way to get the beta version.

Any chance the godot 4 betas can also be available in the github repo? It would be nice to keep up with the godot 4 versions without having to wait an hour to download.

**Comment by Calinou** [comment]
[We don't tag beta releases on GitHub by design](https://github.com/godotengine/godot/issues/68563#issuecomment-1312560814), so it's not possible to upload beta releases on GitHub.
